css/src/components/form/help.scss

```diff
@@ -0,0 +1,35 @@
+$help-color: $text !default;
+$help-error-color: $danger !default;
+$help-warning-color: $warning-active !default;
+$help-success-color: $score-high !default;
+
+$help-font-size: $font-size-8 !default;
+$help-sm-font-size: $font-size-9 !default;
+$help-lg-font-size: $font-size-7 !default;
+
+.help {
+	display: block;
+	margin-block-start: 0.5rem;
+	color: $help-color;
+	font-size: $help-font-size;
+
+	&.help-error {
+		color: $help-error-color;
+	}
+
+	&.help-warning {
+		color: $help-warning-color;
+	}
+
+	&.help-success {
+		color: $help-success-color;
+	}
+
+	&.help-sm {
```

Comment:
remove `-sm`/`-lg` in favor of `.font-size-` atomics?

Reply:
I see your point here, however, I think I would prefer to keep these modifiers. Here is why - initially help's font size is smaller compared to the input/textarea components, so I think it would be much easier to have these two modifiers in favor of input's/textarea's size modifiers so the user doesn't have to guess which exact `.font-size-` class to use to keep the proportions between the form's interactive component and the help text. Let me know if that makes sense to you.

Reply:
I think ideally we just provide a single default and that's what's used most of the time. The .help-<size> modifiers seem extraneous to me. I also don't think we'd need to document .font-size-<size> usage as it relates to this.

Reply:
Updated :)